backup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+BACKUP_DIR="backup-$(date +%H%M_%d%m%Y)"
```

Comment:
minor nit: %Y%m%d_%H%m would provide filenames that sort by date when doing 'ls'

Reply:
please fix example in README.md also.